AGENTS.md

```
@@ -263,12 +328,22 @@ Small Batch (1-2 weeks)
 EOF
 ```
 
-### Workflow
+### The Path from Idea to Built
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)